woocommerce/woocommerce
# Remove title from checkout page

### Changes proposed in this Pull Request:

Remove the title from the Checkout template so that it does no longer appear on the checkout page in the frontend. 

Closes #47509.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `WP Admin » Apperances » Editor » Pages » Checkout`.
2. Verify that the checkout page does not display the `Checkout` title. 
3. Go to `WP Admin » Apperances » Editor » Templates » WooCommerce » Page: Checkout`.
4. Reset customisations, if there are any.
5. Verify that the checkout template does not display the `Checkout` title. 
6. Go to the frontend.
7. Add a product to the cart.
8. Go to the checkout page.
9. Verify that the checkout page does not display the `Checkout` title. 
10. Verify that the site title on the checkout page has the same bottom and top padding as the site title on the cart page.

<table>
<tr>
<td valign="top">Before:
<br><br>
<img width="1280" alt="Screenshot 2024-05-16 at 13 56 50" src="https://github.com/woocommerce/woocommerce/assets/3323310/e9746b1b-86be-4f7f-9287-16d4fe075f8f">
</td>
<td valign="top">After:
<br><br>
<img width="1280" alt="Screenshot 2024-05-16 at 13 57 18" src="https://github.com/woocommerce/woocommerce/assets/3323310/732f808d-c575-4844-a6cb-c92a4d362313">
</td>
</tr>
</table>

11. Go to `/wp-admin/edit.php?post_type=page`.
12. Create a test page and add the checkout shortcode to it:

```
[woocommerce_checkout]
```

13. Go to `/wp-admin/admin.php?page=wc-settings&tab=advanced`.
14. Define the test page with the checkout shortcode as the default checkout page.
15. Go back to the frontend.
16. Go to the checkout page with the checkout shortcode.
17. Verify that the checkout page does not display the `Checkout` title. 

<table>
<tr>
<td valign="top">Before:
<br><br>
<img width="1278" alt="Screenshot 2024-05-17 at 15 22 43" src="https://github.com/woocommerce/woocommerce/assets/3323310/1315b82b-923e-4017-aeb9-c9b51f27d45b">
</td>
<td valign="top">After:
<br><br>
<img width="1278" alt="Screenshot 2024-05-17 at 15 23 11" src="https://github.com/woocommerce/woocommerce/assets/3323310/9536cf21-fe26-4b5a-a2f1-fa04f07de2ab">
</td>
</tr>
</table>

### Changelog entry


<details>

#### Significance




#### Type




#### Message 

Remove title from checkout page

#### Comment 

</details>
